db/migrate/20240619061241_update_reaction_variations.rb

```diff
@@ -0,0 +1,41 @@
+class UpdateReactionVariations < ActiveRecord::Migration[6.1]
+  def up
+    Reaction.all.each do |reaction|
+      variations = reaction.variations
+      variations.each do |variation|
+        %w(startingMaterials products reactants solvents).each do |key|
+          variation[key].each do |_, material|
+            next unless material['value'] && material['unit']
+
+            material['mass'] = { 'value' => material['value'] / 1000, 'unit' => 'g' } # Standard unit was mg, convert to g.
+            material.delete('value')
+            material.delete('unit')
+            material['amount'] = { 'value' => 0, 'unit' => 'mol' }
+            material['volume'] = { 'value' => 0, 'unit' => 'l' }
+          end
+        end
+      end
+      reaction.update(variations: variations)
```

Comment:
use `update_columns` so to not change the timestamp updated_at and what the user expect from the list sorting. Only user changes should be reflected in this column.